include/deal.II/grid/grid_tools.h

```diff
@@ -157,10 +158,27 @@ namespace GridTools
    * get the same result using <code>cell-@>measure()</code>, but this
    * function also works for cells that do not exist except that you make it
    * up by naming its vertices from the list.
+   *
+   * Instantiations of this function are available for the <tt>dim !=
+   * spacedim</tt> cases, but these just raise exceptions. They are only
+   * provided as an aid for generic programming.
+   */
+  template <int dim, int spacedim>
```

Comment:
@tamiko -- it's actually only about 95% :-) (See https://docs.google.com/forms/d/1K2nhtfGuf15Cdw6E4_nbPk82zS9-XNfuRqOjFBWkp7E/viewanalytics )